web/vibe/web/common.d

```diff
@@ -492,25 +492,70 @@ package struct MethodAttribute
 	alias data this;
 }
 
+
+/** Uses a custom serializer for the method return value.
+
+	Instead of using the default serializer (JSON), this allows to define
+	custom serializers. Multiple serializers can be specified and will be
+	matched against the `Accept` header of the HTTP request.
+*/
+alias resultSerializer(alias serialize, alias deserialize, string content_type)
+	= ResultSerializer!(serialize, deserialize, content_type);
```

Comment:
Can you add a `Params` section ?
Also maybe name them `serializeFn` and `deserializeFn` to make things a bit clearer ?

Reply:
I thought about it, but was afraid ddoc would complain because they are not function parameters, but I'm probably wrong. I'll try it out.

Reply:
Yeah DDOC supports that use case.